build-logic/performance-testing/src/main/kotlin/gradlebuild/performance/tasks/BuildCommitDistribution.kt

```diff
@@ -123,7 +123,11 @@ abstract class BuildCommitDistribution @Inject internal constructor(
     private
     fun copyToFinalDestination(checkoutDir: File) {
         val baseVersion = commitBaseline.get().substringBefore("-")
-        checkoutDir.resolve("subprojects/distributions-full/build/distributions/gradle-$baseVersion-bin.zip").copyTo(commitDistribution.asFile.get(), true)
+        val distribution = checkoutDir.resolve("subprojects/distributions-full/build/distributions/gradle-$baseVersion-bin.zip")
+        if (!distribution.isFile) {
+            throw IllegalStateException("${distribution.absolutePath} doesn't exist. Did you set the wrong base version?\n${distribution.parentFile.list()?.joinToString("\n")}")
```

Comment:
Could we do this even somewhere else earlier in a test in quick feedback which would check that the commit in `gradle.properties` has the correct base version? Or even get rid of the need to specify the base version and detect it automatically?